charts/newrelic-infrastructure-v3/templates/NOTES.txt

```diff
@@ -13,6 +13,149 @@ Those components are needed to have the full experience on NROne kubernetes expl
 
 {{- end }}
 
+{{- if .Values.nodeSelector }}
+
+###########
+# WARNING #
+###########
+
+You have specified "nodeSelector" into values, however the new version of the chart
+support different nodeSelectors per each component, therefore the value specified has been ignored.
+
+Please use ksm.nodeSelector, controlPlane.nodeSelector and kubelet.nodeSelector.
+
+{{- end }}
+
+{{- if .Values.nodeAffinity }}
+
+###########
+# WARNING #
+###########
+
+You have specified "nodeAffinity" into values, however the new version of the chart
+support different nodeAffinity options per component, therefore the value specified has been ignored.
+
+Please use ksm.affinity.nodeAffinity, controlPlane.affinity.nodeAffinity and kubelet.affinity.nodeAffinity.
+
+{{- end }}
+
+{{- if .Values.resources }}
+
+###########
+# WARNING #
+###########
+
+You have specified the "resources" into values, however the new version of the chart
+support different resources per component, therefore the value specified has been ignored.
+
+Please use ksm.resources, controlPlane.resources and kubelet.resources.
+
+{{- end }}
+
+{{- if .Values.tolerations }}
+
+###########
+# WARNING #
+###########
+
+You have specified "tolerations" into values, however the new version of the chart
+support different tolerations per component, therefore the value specified has been ignored.
+
+Please use ksm.tolerations, controlPlane.tolerations and kubelet.tolerations.
+
+{{- end }}
```

Comment:
Same as above

Reply:
tolerations are already present in default. We can give precedence to legacy value.
I am not totally sure we should give precedence to an "old" config that was thought for a different object..
WDYT?

Reply:
I kept it as unsupported for now